cmd/unused/main.go

```diff
@@ -75,5 +77,9 @@ func main() {
 
 	checker := newChecker(mode)
 	l := unused.NewLintChecker(checker)
-	lintutil.ProcessArgs("unused", l, flag.Args())
+	args := flag.Args()
+	if len(fTags) > 0 {
```

Comment:
Style-wise, I'd make the suggestion to express this as `if fTags != "" {`. Here's my [rationale](https://dmitri.shuralyov.com/idiomatic-go#empty-string-check).

I really thought the code said `if len(args) > 0 {` at first, because it seems so much more natural to check length of slices with `len`.